src/components/Dropdown.js

```diff
@@ -80,29 +85,60 @@ export default class Dropdown extends React.Component {
     }
   }
 
+  handleKeys = event => {
+    if (this.refs.list) this.refs.list.handleKeys(event)
+  }
+
+  chooseChild = choice => {
+    const link = findChildLink(findDOMNode(this.refs[choice.ref]))
```

Comment:
isn't choice itself a DOM node? can you click it directly rather than having to do `findChildLink`?